l2gersync/evm_downloader_legacy.go

```diff
@@ -172,21 +191,17 @@ func (d *downloaderFEP) getGERsFromIndex(
 		if err != nil {
 			return nil, fmt.Errorf("failed to retrieve l1 info tree leaf for index=%d: %w", i, err)
 		}
-		gers = append(gers,
-			&GlobalExitRootInfo{
-				L1InfoTreeIndex: i,
-				GlobalExitRoot:  info.GlobalExitRoot,
-			})
+		gers = append(gers, newGlobalExitRootInfo(info.GlobalExitRoot, i, blockNum, 0))
```

Comment:
```suggestion
		gers = append(gers, newGlobalExitRootInfo(info.GlobalExitRoot, i, blockNum, info.BlockPosition))
```

Reply:
Very good point, I have reused the block number and block position from L1InfoTreeLeaf (which is created by indexing the `UpdateL1InfoTree`/`UpdateL1InfoTreeV2` events). Also changed the logic a bit, so it doesn't report the empty blocks in l2 ger syncers (which I believe doesn't make sense) https://github.com/agglayer/aggkit/pull/743/commits/1deab4898a211274d3302c4b7c09fcd583c82242